data-raw/example_model_output.R

```diff
@@ -5,7 +5,8 @@
 library(hubUtils)
 hub_path <- "../example-complex-forecast-hub"
 example_model_output <- hubUtils::connect_hub(hub_path) |>
-    dplyr::collect()
+    dplyr::collect() |>
```

Comment:
@lshandross I made these changes to help re-organize the target data, but that may be changed later anyways.